packages/angular/package.json

```diff
@@ -15,9 +15,9 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@angular/common": "10.x || 11.x || 12.x || 13.x || 14.x",
-    "@angular/core": "10.x || 11.x || 12.x || 13.x || 14.x",
-    "@angular/router": "10.x || 11.x || 12.x || 13.x || 14.x",
+    "@angular/common": "10.x || 11.x || 12.x || 13.x || 14.x || 15.x",
```

Comment:
Aside, maybe it is easier to make this: `>= 10 && <= 15`? As the list gets longer and longer 😅

Reply:
Good point 😅